postcodeIO.py

```diff
@@ -0,0 +1,35 @@
+from flask import Flask, request, jsonify
+
+app = Flask(__name__) 
+
+@app.route('/getHouses', methods=['POST'])
+def postcodesIO():
+    frontData = request.get_json()
+    latitude = frontData['lat']
+    longitude = frontData['longlngitude']
```

Comment:
is the key `longlngitude` correct, considering the other one is simply `lat`?

Reply:
I have changed this to lon in my revision as per Jon's specification.